Tools/AP_Bootloader/board_types.txt

```diff
@@ -290,6 +290,7 @@ AP_HW_VIMDRONES_PERIPH               1407
 AP_HW_PIXHAWK6X_PERIPH               1408
 AP_HW_CUBERED_PERIPH                 1409
 AP_HW_RadiolinkPIX6                  1410
+AP_HW_JHEMCU-H743HD                  1411
```

Comment:
board type already taken...you should create a separate PR just for this with an open number and remove this from this PR....it can be merged without a DevCall review while this PR is being readied for DevCall review

Reply:
How do I know which board id is taken? Everything below 19999? I didn't see a comment reserving 1411.

I'm in the process of testing the board. What else should I test for, other than making sure all pinouts work?

I'm not affiliated with the board's company, I'll notify them, see if they want to send their boards out.

Reply:
> How do I know which board id is taken? Everything below 19999? I didn't see a comment reserving 1411.

I also don't see the ID as already taken, @Hwurzburg 

But, @SCRainbow I do suggest you take Henry's advice and PR your ID (not a comment, just take it)

Reply:
sorry, my eyes deceived me...so just PR it and it will be reserved until you are ready to have this merged..

Reply:
@peterbarker https://github.com/ArduPilot/ardupilot/pull/26841 PR for board id

Reply:
@Hwurzburg All done, I think. Should be ready for DevCall.

Reply:
I am creating a testing doc for bd ports..here is an early draft of things to check:
the test setup should have a receiver and GPS/Compass attached to the board
load the firmware and test:
-the board boots and connects to a GCS via USB (Mission Planner, QGC, MAVProxy,etc.)
-the GPS and Compass is recognized.
-the RC is recognized and RC input follows the TX
t-he pitch and roll reports (ie HUD horizon) follows autopilot movement correctly (ie IMU is oriented properly and working)..isolate and test each IMU using INSx_USE params
-board arms with default arming checks(may need to force arm if GPS is indoors)
-attach a battery either directly if onboard power sensors or via external power module and make sure voltage is correctly displayed and current indication present
-attach a test servo to each output after setting the output to a normal function like elevator, and exercise each one with TX while in MANUAL mode to check output functionality.
-test DHot and BDShot capable outputs with BDShot ESC
-move the GPS to each UART output after setting all other UART protocols to NONE and the tested UART to GPS to ensure the UART is functioning

Reply:
I have tested some of these, and I'm in the process of testing more.
I used radio rx for uart testing.
I used a power supply to test the voltage and current readouts, current seemed off at the moment, I'll test more with motors and rotors attached.

Reply:
Just tested this FC and this code on my new copter build 
It's fly!!! 
ELRS Rx / GPS+compass / Analog Video / 4-in-1 esc / 3DRadio 
Everything work as expected